### DISCUSSION TITLE
security: SecKeychainItemImport: Unknown format in import.

### DISCUSSION THREAD

**Discussion by LayMui** [fastlane/fastlane#20188] [Q&A]
when I run fastlane match adhoc
I got this error

what cause this?

**Comment by Thompsons21** [comment]
I can - from Keychain Access app. It asks for the pass and it puts it into keychain, with an red x and a "not trusted" remark. When it comes to noErr, this is a simplified code in as little lines as possible.  [Monkey](https://www.monkey.onl/)
